pkg/plugins/trivy/plugin.go

```diff
@@ -1050,6 +1058,10 @@ func (p *plugin) getPodSpecForClientServerMode(ctx trivyoperator.PluginContext,
 	}, secrets, nil
 }
 
+func getUniqueScanResultFileName(name string) (string, error) {
```

Comment:
no need to return error 
```suggestion
func getUniqueScanResultFileName(name string) string {
```